README.md

```diff
@@ -1,6 +1,58 @@
 ![Logo Dark](https://github.com/camunda/issues/blob/main/resources/Logo_White_Github.png#gh-dark-mode-only)
 ![Logo Light](https://github.com/camunda/issues/blob/main/resources/Logo_Black.png#gh-light-mode-only)
 
-# Welcome to Camunda Issues 
+# Welcome to Camunda Public Roadmap 
```

Comment:
Why rename to "Public Roadmap"?

Can we not keep it as "Issues" and explain that is serves 2 purposes:

* Highlight the roadmap
* Give the ability to open generic issues around Camunda Platform, that do not fit into any of the public component repositories?

Reply:
I don't have a strong opinion on this, we can keep the name of the repository as it is.